packages/contracts-bedrock/contracts/echidna/FuzzAddressAliasing.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity 0.8.15;
+
+import { AddressAliasHelper } from "../vendor/AddressAliasHelper.sol";
+
+contract EchidnaFuzzAddressAliasing {
+    bool failedRoundtrip;
+
+    /**
+     * @notice Takes an address to be aliased with AddressAliasHelper and then unaliased
+     *         and updates the test contract's state indicating if the round trip encoding
+     *         failed.
+     */
```

Comment:
Quick explainer of how Echidna properties work:

You provide a target contract to echidna, ie.
```
echidna-test --contract EchidnaFuzzAddressAliasing
```

Echidna will then deploy the contract, and submit sequences of transactions (up to 100 tx per sequence) to any public methods which do not begin with `echidna_`. 

After each sequence it will then call those methods beginning with `echidna_`, which should return a boolean, where `true` means that the invariant has been violated.